grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-grid.directive.ts

```diff
@@ -1,21 +1,36 @@
 import { CdkDropList, moveItemInArray } from '@angular/cdk/drag-drop';
-import { Directive, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { ApplicationRef, Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
 import { Subscription } from 'rxjs';
 import { tap } from 'rxjs/operators';
+import { CustomClrColumnOrderingService } from './custom-clr-column-ordering.service';
+
+export interface ColumnOrderChangedEvent {
+  column: any;
```

Comment:
This could be a generic `ColumnOrderChangedEvent<TColumn = any>` so that you can use `ColumnOrderChangedEvent<Column>` in the event handler method signature instead of needing the `as Column` type assertion.

Reply:
fixed